### PR TITLE
Six IDE and Code Editing Resources Added

### PR DESCRIPTION
**Pull request by rajputpritesh1**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1580,6 +1581,7 @@ Update Time, five active automations, webhooks.
    * [Binder](https://mybinder.org/) - Turn a Git repo into a collection of interactive notebooks. It is a free public service.
    * [BlueJ](https://bluej.org) — A free Java Development Environment designed for beginners, used by millions worldwide. Powered by Oracle & simple GUI to help beginners.
    * [Bootify.io](https://bootify.io/) - Spring Boot app generator with custom database and REST API.
+   * [Brackets](http://brackets.io/) - Brackets
```
Comment (ripienaar) [inline README.md:1584]: we'll need a better description than one word

File: README.md [hunk 2]
```diff
@@ -1590,6 +1592,7 @@ Update Time, five active automations, webhooks.
    * [codesandbox.io](https://codesandbox.io/) — Online Playground for React, Vue, Angular, Preact, and more.
    * [Components.studio](https://webcomponents.dev/) - Code components in isolation, visualize them in stories, test them, and publish them on npm.
    * [Eclipse Che](https://www.eclipse.org/che/) - Web-based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community-driven. An online instance hosted by Red Hat is available at [workspaces.openshift.com](https://workspaces.openshift.com/).
+   * [Emacs](https://www.gnu.org/software/emacs/) -Emacs
```
Comment (ripienaar) [inline README.md:1595]: I dont think we'll be adding this one, its in every distro, people know it.  We list here more out of the box things like projects thats not so ingrained, I am certainly not looking to list ever single open source project here.

Same for vi of course

File: README.md [hunk 3]
```diff
@@ -1617,6 +1621,7 @@ Update Time, five active automations, webhooks.
    * [wakatime.com](https://wakatime.com/) — Quantified self-metrics about your coding activity using text editor plugins, limited plan for free.
    * [WebComponents.dev](https://webcomponents.dev/) — In-browser IDE to code web components in isolation with 58 templates available, supporting stories, and tests.
    * [PHPSandbox](https://phpsandbox.io/) — Online development environment for PHP.
+   * [Xcode For MacOS](https://developer.apple.com/xcode/) -Xcode For MacOS
```
Comment (ripienaar) [inline README.md:1624]: also pretty much just part of the OS and generally not an option to not use in reality, so i'd rather not add a redundant entry to an already massive list

File: README.md [hunk 4]
```diff
@@ -1824,6 +1829,7 @@ Update Time, five active automations, webhooks.
   * [ReqBin](https://www.reqbin.com/) — Post HTTP Requests Online. Popular Request Methods include GET, POST, PUT, DELETE, and HEAD. Supports Headers and Token Authentication. Includes a basic login system for saving your requests.
   * [Smartcar API](https://smartcar.com) - An API for cars to locate, get fuel tank, battery levels, odometer, unlock/lock doors, etc.
   * [snappify](https://snappify.com) - Enables developers to create stunning visuals. From beautiful code snippets to fully fletched technical presentations. The free plan includes up to 3 snaps at once with unlimited downloads and 5 AI-powered code explanations per month.
+  * [Sublime Text](https://www.sublimetext.com/) - Sublime Text
```
Comment (ripienaar) [inline README.md:1832]: Again, cant just add things with their name as description.  It's also in the wrong category.

**Review by ripienaar — CHANGES_REQUESTED** [review]

**Review by rajputpritesh1 — COMMENTED** [review]
All Changes Done Any Other Query ??

File: README.md [hunk 5]
```diff
@@ -1609,14 +1611,16 @@ Update Time, five active automations, webhooks.
    * [Replit](https://replit.com/) — A cloud coding environment for various program languages.
    * [SoloLearn](https://code.sololearn.com) — A cloud programming playground well-suited for running code snippets. Supports various programming languages. No registration is required for running code, but it is necessary when saving code on their platform. Also offers free courses for beginners and intermediate-level coders.
    * [stackblitz.com](https://stackblitz.com/) — Online/Cloud Code IDE to create, edit, & deploy full-stack apps. Support any popular NodeJs-based frontend & backend frameworks. Shortlink to create a new project: [https://node.new](https://node.new).
+   * [Sublime Text](https://www.sublimetext.com/) - Sublime Text is a popular, versatile, and highly customizable text editor used for coding and text editing tasks.
+   * [Vim](https://www.vim.org/) - Vim
```
Comment (ripienaar) [inline README.md:1615]: Please remove vim

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -630,6 +630,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 ## Security and PKI
 
   * [alienvault.com](https://www.alienvault.com/open-threat-exchange/reputation-monitor) — Uncovers compromised systems in your network
+  * [Atom](https://atom.io/) - Atom
```
Comment (ripienaar) [inline README.md:633]: So sorry, missed you added this one - but again its just one word.  It's also just word and its EOL so should be removed.

**Review by rajputpritesh1 — COMMENTED** [review]
Done

**Comment by ripienaar** [comment]
> Done

Sorry, did you not see the editor is EOL and it should be removed?

Please, reviewing PRs takes time for people, please read the review feedback.

**Review by rajputpritesh1 — COMMENTED** [review]
Description Added For Atom

File: README.md (hunk 5 above)
Comment (rajputpritesh1) [reply README.md]: done

**Comment by ripienaar** [comment]
Thank you

**Review by rajputpritesh1 — COMMENTED** [review]
I Think Now All Done

**Comment by rajputpritesh1** [comment]
Thank You!

**Pull request merged**